src/xargs/mod.rs

```diff
@@ -818,6 +847,25 @@ fn do_xargs(args: &[&str]) -> Result<CommandResult, XargsError> {
                 .help("Be verbose")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::REPLACE)
+                .long(options::REPLACE)
+                .short('i')
+                .num_args(0..=1)
+                .require_equals(true)
+                .value_parser(clap::value_parser!(String))
+                .help(
+                    "Replace R in INITIAL-ARGS with names read from standard input; \
+                    if R is unspecified, assume {}",
+                ),
+        )
+        .arg(
+            Arg::new(options::REPLACE_I)
+                .short('I')
+                .num_args(1)
+                .hide(true)
```

Comment:
Why do you hide the option? GNU `xargs` lists it with `--help`.
```suggestion
                .help("same as --replace=R")
                .value_name("R")
```